srcpkgs/googlemaps/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'googlemaps'
+pkgname=googlemaps
+version=.0.0.0.2
+revision=1
+archs="i686 x86_64 x86_64-musl aarch64 aarch64-musl"
```

Comment:
Should be able to build for all archs, so you don't need this line.

Reply:
Done